src/utils/config.ts

```
@@ -0,0 +1,17 @@
+export const IsCustomDocumentLoaderEnabled = (): boolean => {
+  const flag = process.env.ENABLE_CUSTOM_DOCUMENT_LOADER ?? 'false'
+  const isCustomDocumentLoaderEnabled = flag.toLowerCase() === 'true'
+
+  if (isCustomDocumentLoaderEnabled) {
+    if (!process.env.DEPRECATED_DOMAIN || !process.env.CURRENT_DOMAIN) {
+      console.error(
```

ESLint

[error] Unexpected console statement.

(no-console)